frontend/src/Components/Profile.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from 'react'
+import { Login } from 'Components/Login'
+
+const URL = 'https://auth-login-project.herokuapp.com/users'
+
+export const Profile = (loggedInUser) => {
+  // loggedInUser = loggedInUser.loggedInUser
+  const [userId, setUserId] = useState(0)
+  const [accessToken, setAccessToken] = useState('')
+  console.log(loggedInUser)
+
+  const [logout, setLogout] = useState(false)
```

Comment:
this is not used anywhere? was the plan to do something with it here or did you just forgot to remove it?